README.md

```diff
@@ -377,7 +377,7 @@ Fire DOM events.
 ```javascript
 // <button>Submit</button>
 fireEvent(
-  getElementByText('Submit'),
+  getByText('Submit'),
```

Comment:
Nice! This should also receive a `container` as the first argument. Could you add that?

Reply:
Can do!